src/main/java/sc/fiji/bdvpg/scijava/services/SourceAndConverterBdvDisplayService.java

```diff
@@ -166,37 +167,37 @@ public void makeInvisible(SourceAndConverter sac) {
      * It mimicks or copies the functions of BdvVisTools because it is responsible to
      * create converter, volatiles, convertersetups and so on
      * @param sac
-     * @param bdvh
+     * @param bdvHandle
      */
-    public void show(BdvHandle bdvh, SourceAndConverter sac) {
+    public void show(BdvHandle bdvHandle, SourceAndConverter sac) {
         // If the sourceandconverter is not registered, register it
-        if (!bss.isRegistered(sac)) {
-            bss.register(sac);
+        if (!bdvSourceAndConverterService.isRegistered(sac)) {
+            bdvSourceAndConverterService.register(sac);
         }
 
         // Escape if the sourceandconverter is already shown is this BdvHandle
-        if ( sacToBdvHandleRefs.get(sac)!=null) {
-            if ( sacToBdvHandleRefs.get(sac).stream().anyMatch( bdvhr -> bdvhr.bdvh.equals(bdvh))) {
+        if ( sacToBdvRefs.get(sac)!=null) {
+            if ( sacToBdvRefs.get(sac).stream().anyMatch( bdvhr -> bdvhr.bdv.equals(bdvHandle))) {
                 return;
             }
         }
 
         // Stores in which BdvHandle the sourceandconverter will be displayed
-        if (!bdvHandleToSacs.containsKey(bdvh)) {
-            bdvHandleToSacs.put(bdvh, new ArrayList<>());
+        if (!bdvHandleToSacs.containsKey(bdvHandle)) {
+            bdvHandleToSacs.put(bdvHandle, new ArrayList<>());
         }
-        bdvHandleToSacs.get(bdvh).add(sac);
+        bdvHandleToSacs.get(bdvHandle).add(sac);
 
-        bdvh.getViewerPanel().addSource(sac);
-        bdvh.getSetupAssignments().addSetup(getConverterSetup(sac));
+        bdvHandle.getViewerPanel().addSource(sac);
+        bdvHandle.getSetupAssignments().addSetup(getConverterSetup(sac));
 
         // Stores where the sourceandconverter is displayed (BdvHandle and index)
-        BdvHandleRef bhr = new BdvHandleRef(bdvh, bdvh.getViewerPanel().getState().numSources());
-        if (!( sacToBdvHandleRefs.containsKey(sac))) {
+        BdvHandleRef bhr = new BdvHandleRef(bdvHandle, bdvHandle.getViewerPanel().getState().numSources());
```

Comment:
This sort of refactoring is a bit annoying for the reviewer. Let's try to refactor less, and in the master. It will be more straightforward for reviews to see what are the change that actually matter

Reply:
True! Let's try to avoid this.